upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -93,6 +93,8 @@ var (
 	CloudupModels = []string{"proto", "cloudup"}
 	// OldestSupportedKubernetesVersion is the oldest kubernetes version that is supported in Kops
 	OldestSupportedKubernetesVersion = "1.9.0"
+	// OldestRecommendedKubernetesVersion is the oldest kubernetes version that is not deprecated in Kops
+	OldestRecommendedKubernetesVersion = "1.10.0"
```

Comment:
Maybe it's just me, but I would not recommend any k8s version where the default etcd is not 3.x. Anyone having to got through the upgrade from 2.x to 3.x will be pretty annoyed.
https://github.com/kubernetes/kops/blob/298f79659a2bd9c1d6548c6c95b335b5e29e6466/pkg/model/components/etcdmanager/options.go#L60-L61

Reply:
The current policy as I understand it is to remove support from 1.8 and earlier in kops 1.18, then do rolling removals in subsequent versions. That means removing support from 1.9 in kops 1.19. There was generalized talk about narrowing the gap in future releases, but no specific decisions there have been made.

Reply:
I don't have anything against supporting these versions, but the name says "recommended" version. I think "non-deprecated" is not quite the same as "recommended".

Reply:
I am not able to find a better word for "non-deprecated". The code uses "recommended" for "non-deprecated" later in the file, where it's dealing with channels.

Reply:
Maybe we can use "legacy" here.

Reply:
The word "recommended" is not made visible to the user. The constant only demarcates what is not described as "deprecated".

Reply:
Yeah, if & when we come up with better words we can rename this constant without affecting anything user-visible!